game/missiongenerator/aircraft/waypoints/recoverytanker.py

```diff
@@ -10,6 +10,11 @@ class RecoveryTankerBuilder(PydcsWaypointBuilder):
     def add_tasks(self, waypoint: MovingPoint) -> None:
 
         assert self.flight.flight_type == FlightType.REFUELING
+
+        waypoint.add_task(
```

Comment:
Add a code comment with a link to the bug please. This is pretty non obvious DCS behavior :)

Reply:
Done :)